.storybook/config.js

```diff
@@ -4,6 +4,30 @@ import {addReadme} from 'storybook-readme';
 import {withA11y} from '@storybook/addon-a11y';
 import withThemeSwitcher from './addons/theme-switcher-addon';
 
+const customViewports = {
+  Mobile: {
+    name: 'Mobile',
+    styles: {
+      width: '24rem',
```

Comment:
Manually added these from the default theme. Subtracted 1 rem in order to get the correct viewport with the addon. 25rem wouldn't actually show the correct view for the media query.

Reply:
Can you import these values directly from our tokens package?

Reply:
We can, but this add on creates an iframe with the width, so our media queries don't line up. The iframe with is 25rem using our smallest theme breakpoint, so `@media screen and (min-width: 25rem)` is still enabled in the styles. See screenshot:

![image](https://user-images.githubusercontent.com/1350081/75269886-e3754800-57b6-11ea-82e5-7cd50bae9377.png)

If I set to 24rem, it works as expected. I could run some js to subtract 1 from our theme breakpoints. Thoughts?

Reply:
I'd rather not have 2 sources of truth for theme values anywhere, so I'd lean towards some computation based on those values.  However the bigger question is: is this addon helpful if the breakpoints aren't the ones users expect? How big of a deal is that 1rem offset?

Reply:
Not a big deal to calculate that. Just pushed up that change.

Reply:
Screenshot showing expected mobile view:
![image](https://user-images.githubusercontent.com/1350081/75274036-55518f80-57bf-11ea-9629-0679b9c2dfbb.png)